### ISSUE TITLE
[REMOTO] Estagio para construção de UI/UX utilizando Inteligência de Máquina

### ISSUE THREAD

**Issue by ktfth** [frontendbr/vagas#1488]
Empoderando o desenvolvimento de UI/UX através da Inteligência de Máquina

## Descrição da vaga

Projeto para construção de um modelo de inteligência de máquina
que aprenda compor uma interface utilizando uma super inteligência.

## Local

Remoto mas o projeto esta em São Paulo

## Benefícios

- Recomendação através da reputação alcançada através do projeto
- Prêmio para produção do conteúdo em dinheiro
- Participação para fundação do produto

#### Diferenciais

- Flexibilidade para sugestões até mesmo de como pode se divertir ao envolver-se
  com o projeto

## Requisitos

Interesse em produzir especificações programaticas de interfaces de usuário.

**Obrigatórios:**

- Ter o desejo de aprender algo novo

**Desejáveis:**

- Tensorflow
- Keras
- Angular

**Diferenciais:**

- projetos opensource

## Contratação

Aberto para negociações de forma livre

## Nossa empresa

https://fth.sh

## Como se candidatar

Por favor envie um email para kaeyosthaeron@gmail.com com seu CV anexado - enviar no assunto:
Inteligência artificial para interfaces

## Labels

- Alocado
- Estágio
- Freela
- PJ
- Remoto
- Sênior
